### PR TITLE
Update versions and CHANGELOG.md

### PR DESCRIPTION
**Pull request by Walingar**

### CODE REVIEW

File: ci/compose-uber-jar/gradle.properties [hunk 1]
```diff
@@ -1,2 +1,2 @@
-compose.version=1.5.1
+compose.version=1.5.10-beta01
```
Comment (pjBooms) [inline ci/compose-uber-jar/gradle.properties:1]: Do we need to change Compose version to beta in samples? @igordmn

Comment (igordmn) [reply ci/compose-uber-jar/gradle.properties:1]: We can change them, but we should create `support/1.5.10` branch, and merge the PR there.

`release/1.5.10` won't be merged back to master, `support/1.5.10` will be merged back to master (after the release).

Comment (igordmn) [reply ci/compose-uber-jar/gradle.properties:1]: Changelog itself we should merge to master, not to `support/1.5.10`

Comment (Walingar) [reply ci/compose-uber-jar/gradle.properties:1]: I thought that `support/1.5.10` is not needed. I mean I'll cherry-pick commits from here to master directly when beta will be released

Comment (Walingar) [reply ci/compose-uber-jar/gradle.properties:1]: I can keep examples with 1.5.1, for me it's ok

**Comment by igordmn** [comment]
The target branch should be "master"

**Comment by Walingar** [comment]
Sure, will create new one

**Pull request closed without merging**